docs/modules/ROOT/pages/utils/_class_hashes.adoc

```diff
@@ -1,12 +1,13 @@
 // Version
-:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/2.5.3[cairo 2.5.3]
+:class-hash-cairo-version: https://crates.io/crates/cairo-lang-compiler/v2.5.3[cairo v2.5.3]
```

Comment:
We need to remove the v from the workflow

Reply:
From the whole workflow? Can't we just ignore this doc?

Reply:
I only question this because it seems more likely that unintended things may change by omitting the v. Maybe it's not a big deal idk

Reply:
Is just for the script, I updated it already in the other PR

Reply:
The PR only touches 2 lines, so it should be fairly easy to review.

Reply:
My bad, I thought you also meant with the replace version in files job as well